api/mfx_dispatch/windows/include/mfx_dispatcher.h

```diff
@@ -218,4 +218,76 @@ FUNCTION_DESCRIPTION APIFunc[eVideoFuncTotal];
 
 extern const
 FUNCTION_DESCRIPTION APIAudioFunc[eAudioFuncTotal];
+
+
+#if (defined(_WIN64) || defined(_WIN32)) && (MFX_VERSION >= MFX_VERSION_NEXT)
```

Comment:
Why this API is made _windows_ dispatcher specific. I think it makes sense to consider declaring it on mediasdk API level otherwise it is not quite clear whether this API is official or not?

Reply:
Currently we have only windows implementation, so we will have those defines for some time anyway. It is logical that those functions should be implemented on dispatcher layer (or in some separate lib operating on same layer than dispatcher), so technically it is not part of MSDK library.

Reply:
From precedent point of view, doc/mediasdk-man.pdf already define API in terms of lib+dispatcher. I.e. MFXInit would behave slightly differently w/o dispatcher. Absence of Linux implementation is a clear gap.

Reply:
exposed to MSDK API